uber/utils.py

```diff
@@ -542,7 +542,15 @@ def genpasswd():
 # ======================================================================
 
 def redirect_to_allowed_dept(session, department_id, page):
+    error_msg = 'You have been given admin access to this page, but you are not in any departments that you can admin. ' \
+                'Please contact STOPS to remedy this.'
+                
+    if c.DEFAULT_DEPARTMENT_ID == 0:
+        raise HTTPRedirect('../accounts/homepage?message={}'.format(error_msg))
```

Comment:
You don't need the ``.format``, because we extend ``HTTPRedirect`` so that you can just say

```python
raise HTTPRedirect('../accounts/homepage?message={}', error_msg)
```